### ISSUE TITLE
[Screen Reader – Make Code - Publish project] Screen reader does not pass the 'Copied' information after selecting Copy button.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3602]
**User Experience:**
User won't get the purpose of any control or element if screen reader doesn't provide descriptive information of that control and also user won’t be able to take action on such control if screen reader doesn't provide descriptive information which is relevant to that control.

**Note:** User credentials should NOT be included in the bug. 

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Share" option appearing in header section and select it.
6. Navigate to "Publish project" under  "Share Project" dialog and select it.
7. Navigate to all the control appearing like QR Code, Embed(Code, Editor and Simulator).
8. Verify that Screen reader announces the 'Copied' information after selecting Copy button or not.

**Actual Result:**
Screen reader does not pass the 'Copied' information after selecting Copy button under 'Share Project' pop-up.

**Expected Result:**
Screen reader should announce the 'Copied' information after selecting Copy button under 'Share Project' pop-up.

**Note:**
Same issue exists on Firefox + NVDA & Chrome + JAWS.

**WCAG Reference:**
[WCAG 1.3.1 - Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html})

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG1.3.1, #Win10-Edge(Chromium)

[WCAG1.3.1_Publish project_Copied information.zip](https://github.com/microsoft/pxt-microbit/files/5499275/WCAG1.3.1_Publish.project_Copied.information.zip)

**Comment by jwunderl** [comment]
button indicates in screenreader that is a copy button, and when user copies it indicates what value was copied (the url for the project).
